website/src/docs/hotchocolate/defining-a-schema/lists.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Lists"
+---
+
+import { ExampleTabs } from "../../../components/mdx/example-tabs"
+
+GraphQL allows us to return lists of elements from our fields.
+
+```sdl
+type Query {
+  users: [User]
+}
+```
+
+Clients can query list fields like any other field.
+
+```graphql
+{
+  users {
+    id
+    name
+  }
+}
+```
+
+Querying a list field will result in a JSON array containing elements with the specified subselection of fields.
```

Comment:
Is it GQL spec that the response must be JSON? I mean 99% of the time it is, but maybe we should just remove JSON and say it's an array. As technically the execution engine's result could be serialized to XML or something else. Bit nitpicky :P

Reply:
No, the specification only talks about a ordered list. Will change.